docs/console/console-extensibility/activation.md

```diff
@@ -0,0 +1,150 @@
+```json
+{
+  "tenantId": "my-tenant-id",
+  "extensionId": "my-extension-id"
+}
+```
+
+**Body Params**
+```json
+{
+  "contextType": "my-project-id",
+  "contextId": "project",
+  "overrides": []
+}
+```
+
+**Response on success**
+```json
+{
+    "activationId": "my-activation-id"
+}
+```
+
+## Overrides
+
+Overrides allow us to edit the routes of a registered extension graphically applying these changes only for a specific activation. Activations without overrides show the extension's menu item as it was registered with [this API](./registration.md#how-to-register-my-extension), while with overrides there could be graphical differences like a different icon or label.
+
+:::info
+With overrides, it is possible to edit one or more routes of the registered extension, including routes of type `category`. The override on a route will apply only the specified changes and leave the others unchanged, so except for the routeId the other properties are optional.
+:::
+
+**Override properties**
+- `routeId`(_required_): specifies the id of the registered extension's route on which apply the changes
+- `order`
+- `icon`
+- `labelIntl` 
+
+The properties that can be modified by the override coincide with the properties of the routes registered with the extension already explained [here](./registration.md#how-to-configure-correctly-the-extension-menu-item)
+
+### Apply overrides on Backoffice Extension Example
+
+![activate extension with override](./images/activateExtensionWithOverride.png)
+
+With the following example, you can activate the Backoffice extension as done [above](#activate-backoffice-extension-example) applying overrides to change the label of the menu group and the icon of the extension route.
+
+:::warning
+Be careful to not activate the same extension multiple times on the same context if the extension has already been activated with the example above, so remember to change the project id on which to activate the extension again or delete the previous activation (see [here](#deactivate-an-extension)).
+:::
+
+**Path Params**
+```json
+{
+  "tenantId": "my-tenant-id",
+  "extensionId": "my-extension-id"
+}
+```
+
+**Body Params**
+```json
+{
+  "contextId": "project",
+  "contextType": "company",
+  "overrides": [
+    {
+      "routeId": "backoffice-route",
+      "icon": {
+        "name": "PiAcorn"
+      }
+    },
+    {
+      "routeId": "my-menu-group",
+      "labelIntl": {
+        "en": "Overrided Label for menu group",
+        "it": "Etichetta Sovrascritta per gruppo menu"
+      }
+    }
+  ]
+}
+```
+
+**Response on success**
+```json
+{
+    "activationId": "my-activation-id"
+}
+```
+
+<!-- TODO: Explain that an override refers to the route registered with the extension using the routeId (Invite the user to remember the routeId or to use the GET activate extension api to retrieve the registered extension with its routes) -->
+<!-- TODO: Add an image on Console with the added extension showing the overrides  -->
```

Comment:
```suggestion
```

I don't think it's that useful. The reader can easily figure out the result

Reply:
perfect